Rear Rider/Rear Rider/CameraTestView.swift

```diff
@@ -26,19 +29,19 @@ struct CameraTestView: View {
             Text(mlResult)
             
             HStack(spacing: 100) {
```

Comment:
If this `HStack` now only has one element in it I don't think you need the stack, you can just use the `Button`